apps/web/src/views/Swap/AkkaSwap/hooks/useAkkaRouterApi.ts

```diff
@@ -7,7 +7,18 @@ import { Field } from 'state/swap/actions'
 import { useSwapState } from 'state/swap/hooks'
 import useSWR, { Fetcher } from 'swr'
 import { AkkaRouterArgsResponseType, AkkaRouterInfoResponseType, TokenEnum } from './types'
+import { useActiveChainId } from 'hooks/useActiveChainId'
 
+export const setChainName = (chainNumber: ChainId): string => {
+  switch (chainNumber) {
+    case ChainId.BITGERT:
+      return "bitgert"
+    case ChainId.XDC:
+      return "xdc"
```

Comment:
Would be great to not have another mapping that needs to be fitted for new chains. Using chainId always is preferred.